catalog/resource_catalog_workspace_binding.go

```diff
@@ -0,0 +1,66 @@
+package catalog
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func contains_int64(s []int64, e int64) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Is this the right place & name for this function?

Reply:
I can get rid of this function if we change the [`contains`](https://github.com/databricks/terraform-provider-databricks/blob/48e7be82bb1c77d99879b78281e61c335a2b6fae/catalog/update.go#L11-L18) function to use generics. For example,
```go
func contains[T comparable](s []T, e T) bool {
	for _, a := range s {
		if a == e {
			return true
		}
	}
	return false
}
```

Reply:
it makes sense to change that function to use generics - it is only used in a single place within the catalog package

Reply:
Awesome! I can do that in a separate PR to keep the concerns separate?

Reply:
sure, feel free to raise a small PR for this one - should be quick to test & approve

Reply:
Done #2373